### PR TITLE
Add an 'Import from hugging face' modal

### PR DESCRIPTION
**Pull request by agatha197**
### TL;DR

Add a new feature to import from Hugging Face in the VFolderListPage.

### What changed?

1. Modified `config.toml.sample` for indentation fix.
2. Added `ImportFromHuggingFaceModal` component to `react/src/components`.
3. Integrated `ImportFromHuggingFaceModal` in `VFolderListPage`.
4. Added new buttons and toggle functionality to `VFolderListPage.tsx`.
5. Updated language JSON files to include new translations for the Hugging Face import feature.

### How to test?

1. [Remove this line ](https://github.com/lablup/backend.ai-webui/pull/2514/files#diff-65e44191bce624d5e51a52298306cdece45d544cfb5ac1cc5fa4a202ddde6693R122)
2. Navigate to the data page. And click the model or model-store tab.
3. You can check the added modal by clicking an 'Import From Hugging Face' button.

### Screenshots

---

**Checklist:** (if applicable)

- [ ] Mention to the original issue
- [ ] Documentation
- [ ] Minium required manager version
- [ ] Specific setting for review (eg., KB link, endpoint or how to setup)
- [ ] Minimum requirements to check during review
- [ ] Test case(s) to demonstrate the difference of before/after

### CODE REVIEW

**Comment by agatha197** [comment]
* **#2514** <a href="https://app.graphite.dev/github/pr/lablup/backend.ai-webui/2514?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @agatha197 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by yomybaby — APPROVED** [review]
LGTM, but it's better to merge later after finishing upcomming stacks for integrating the API.

File: react/src/hooks/backendai.tsx [hunk 1]
```diff
@@ -80,8 +80,8 @@ export const useResourceSlotsDetails = (resourceGroupName?: string) => {
   const [key, checkUpdate] = useUpdatableState('first');
   const baiRequestWithPromise = useBaiSignedRequestWithPromise();
   const baiClient = useSuspendedBackendaiClient();
-  const { data: resourceSlots } = useTanQuery<{
-    [key: string]: ResourceSlotDetail | undefined;
+  let { data: resourceSlots } = useTanQuery<{
+    [key: string]: ResourceSlotDetail;
```
Comment (yomybaby) [inline react/src/hooks/backendai.tsx]: Please check this change. It seems like an unintended change.

**Review by yomybaby — CHANGES_REQUESTED** [review]

**Review by yomybaby — APPROVED** [review]
LGTM
